src/template.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title><%= htmlWebpackPlugin.options.title %></title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="todo-app">
+        <h2>Today's Todo List</h2>
+        <div class="row">
+          <input
+            type="text"
+            id="input-box"
+            placeholder="add your text"
+            autofocus
+          />
+          <button class="add-btn">add</button>
+        </div>
+        <ul class="list-container"></ul>
+        <button class="clear-all">Clear All</button>
```

Comment:
- [ OPTIONAL ] I suggest you to change the name from 'delete all' to 'delete all checked' as it would be more meaningful because it only removes the checked ones.